package.json

```diff
@@ -35,7 +44,8 @@
     "format": "prettier --write ./src/**/*.js",
     "format:check": "prettier -l ./src/**/*.js",
     "verify-eslint-rules": "eslint --print-config .eslintrc | eslint-config-prettier-check",
-    "ci": "npm run verify-eslint-rules && npm run format:check && npm run lint:check"
+    "ci": "npm run verify-eslint-rules && npm run format:check && npm run lint:check",
+    "wdio": "node -r babel-register ./node_modules/.bin/wdio wdio.conf.js"
```

Comment:
I should remove the _wdio.conf.js_ part.

Reply:
Okay, want to do that, then?